src/site/xdoc/server/config-mailetcontainer.xml

```diff
@@ -77,6 +77,28 @@
       pair of ("processor", "spam").</p>
     
     </subsection>
+
+    <subsection name="Error handling">
+      <p>If an exception is encountered during the execution of a mailet or a matcher, the default behaviour is to
+      process the mail using the <strong>error</strong> processor.</p>
+
+      <p>The <strong>onMailetException</strong> property allow you to override this behaviour. You can specify an other
+      processor than the error one for handling the errors of this mailet. The <strong>ignore</strong> special value also
+      allow to continue processing and ignore the error.</p>
+
+      <p>Moreover, the <strong>onMatcherException</strong> allow you to override matcher error handling logic. You can
```

Comment:
you can remove "logic" from the sentence, I think it makes things easier to read and we don't loose much